src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/InnvilgeVedtak/InnvilgeVedtak.tsx

```diff
@@ -90,6 +95,30 @@ export const InnvilgeVedtak: React.FC<{
     const inntektsperioder = inntektsperiodeState.value;
     const vedtaksperioder = vedtaksperiodeState.value;
 
+    useEffect(() => {
+        if (!vedtakshistorikk?.perioder?.length) return;
+
+        const perioderMedEndretKey = vedtakshistorikk.perioder.map((periode) => {
+            return { ...periode, endretKey: uuidv4() };
+        });
+
+        vedtaksperiodeState.setValue(perioderMedEndretKey);
+
+        // eslint-disable-next-line
+    }, [vedtakshistorikk]);
+
+    useEffect(() => {
+        if (!vedtakshistorikk?.inntekter?.length) return;
+
```

Comment:
Kan vi legge inn en featuretoggle her for å ikke ha funksjonaliteten i prod, men få koden inn i master?

Reply:
Supert, husk å teste med toggle av og på for å sjekke at det oppfører seg normalt 😂